### ISSUE TITLE
Ignored a call to create table

### ISSUE THREAD

**Issue by tpetracca** [palantir/atlasdb#968]
Not sure if this is expected but I'm still seeing the following logs on a stack that was *created* using an atlas service on version 0.11.4:

```
WARN  [2016-09-22 13:11:27,981] com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService: Ignored call to create a table (_metadata) that already existed (case insensitive).
WARN  [2016-09-22 13:11:28,618] com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService: Ignored call to create a table (_timestamp) that already existed (case insensitive).
WARN  [2016-09-22 13:11:28,946] com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService: Ignored call to create a table (_transactions) that already existed (case insensitive).
WARN  [2016-09-22 13:11:29,212] com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService: Ignored call to create a table (sweep.priority) that already existed (case insensitive).
WARN  [2016-09-22 13:11:29,212] com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService: Ignored call to create a table (sweep.progress) that already existed (case insensitive).
WARN  [2016-09-22 13:11:29,413] com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService: Ignored call to create a table (<schema-specific-namespace>.<schema-specific-table>) that already existed (case insensitive).
```

**Comment by clockfort** [comment]
Already fixed in #771 when I got annoyed at all of the logging while I ran everything in my IDE.
(double create table = DEBUG, double drop table is still WARN because that seems weirder)

**Comment by tpetracca** [comment]
@ashrayjain as this relates to schema locks.

Not really in the scope of this issue, but generally I would prefer the logic that builds the "existingTablesLowerCased" happen before the createTablesInternal(Map tablesToCreate) method, so that tablesToCreate only includes things that actually need to be created, thereby minimizing cases of us attempting to grab the schema mutation lock
